packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap

```diff
@@ -1745,8 +1769,8 @@ exports[`test file matches snapshot 1`] = `
           0,
           0,
           0,
-          80,
-          191,
+          72,
+          203,
```

Comment:
I guess this is a side-effect of adding a new dataset to the file.

Reply:
Wait, what ? How adding new datasets changes the value of existing ones ?

Reply:
I think those are basically buffer addresses in the file.

Reply:
Oh got it :+1: